Region/Region.cc

```diff
@@ -520,10 +520,22 @@ casacore::IPosition Region::XyOrigin() {
 const casacore::ArrayLattice<casacore::Bool>* Region::XyMask() {
     // returns boolean mask of xy region
     casacore::ArrayLattice<casacore::Bool>* mask;
+    bool extended_region_ok(false);
 
     if (_xy_region != nullptr) {
         // get extended region
-        auto extended_region = static_cast<casacore::LCExtension*>(_xy_region->toLCRegion(_coord_sys, _image_shape));
+        casacore::LCExtension* extended_region;
+        try {
+            extended_region = static_cast<casacore::LCExtension*>(_xy_region->toLCRegion(_coord_sys, _image_shape));
+            extended_region_ok = true;
+        } catch (casacore::AipsError& err) {
+            std::cerr << err.getMesg() << std::endl;
```

Comment:
we probably don't need to output the error if it's simply a region outside an image. it's not an "error" really, it's just a region that contains no data.